internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -44,9 +44,23 @@ UniqueAliasesAction:
   Properties:
     ServiceToken: !GetAtt UniqueJSONValuesFunction.Arn
     Aliases: !Ref Aliases
+    {{- if .CDNConfig.Static}}
+    AdditionalAlias: {{.CDNConfig.Static.Alias}}
+    {{- end}}
```

Comment:
like written in the previous comment, I wonder if instead of a single string it would be more flexible as a list of strings